src/serviceFacades/analyses.js

```diff
@@ -131,6 +151,31 @@ function getAnalysisPermissions({ analyses }) {
     });
 }
 
+const runningViceJobsFilter = [
+    { field: "status", value: AnalysisStatus.RUNNING },
+    { field: "type", value: AppType.interactive.value },
+    { field: "ownership", value: "mine" },
+];
+
+const RUNNING_VICE_JOBS_QUERY_KEY = [
+    ANALYSES_LISTING_QUERY_KEY,
+    { filter: JSON.stringify(runningViceJobsFilter) },
+];
+
+function useRunningViceJobs({ enabled, onSuccess, onError }) {
+    return useQuery({
+        queryKey: RUNNING_VICE_JOBS_QUERY_KEY,
+        queryFn: getAnalyses,
+        config: {
+            enabled,
+            onSuccess,
+            onError,
+            staleTime: Infinity,
+            cacheTime: Infinity,
```

Comment:
What is the benefit of using infinite cache and stale time for this query?

I would think those settings should be used when API data isn't changing (or at least not changing within a typical user session), or if we are manually invalidating the query somewhere else; but I don't see an invalidation in this PR, and this kind of data seems like it could change fairly often 🤔

Reply:
Oops, yeah, this is leftover from a few different refactorings. Initially, I did have an invalidation that would fire when a new analysis notification would come in.  Then Sriram pointed out that the notification data actually has enough information to update the list without hitting the service again. So for this particular feature, it is only fetched once on login, then updated by parsing notifications.

I do think it's confusing to keep the `staleTime` and `cacheTime` settings here though, like you said - I could make them configurable and add them to the point where `useRunningViceJobs` is actually called for this feature.

Reply:
>  Then Sriram pointed out that the notification data actually has enough information to update the list without hitting the service again. So for this particular feature, it is only fetched once on login, then updated by parsing notifications.

Ah, that's right, I forgot about that feature 👍